docker/local-metadata/docker-compose.yml

```diff
@@ -91,6 +91,7 @@ services:
 
 networks:
   local_app_net:
+    name: ometa_network
```

Comment:
This is helpful when hooking up different docker-compose runs. I needed it when trying to align the services between a compose running OM and another one running Airflow 1.10.15